haystack/nodes/prompt/invocation_layer/hugging_face.py

```diff
@@ -318,14 +309,15 @@ def _extract_torch_dtype(self, **kwargs) -> Optional["torch.dtype"]:
     @classmethod
     def supports(cls, model_name_or_path: str, **kwargs) -> bool:
         task_name: Optional[str] = kwargs.get("task_name", None)
+        if not torch_and_transformers_import.is_successful():
+            return False
```

Comment:
Should we perhaps remove those two lines again? Considering the following lines work without the inference install, we would get a more meaningful error message without those lines.

Reply:
Hmm, right. Let me try

Reply:
You are right. We don't need it. I tried four different options: 
- with and without inference
- hf and hf inference layer

Everything works as expected. In addition, as you suggest, the error is more meaningful in cases when the user intends to use hugging_face.py layer yet forgot to install torch.